enhancements/kube-apiserver/audit-policy.md

```diff
@@ -40,8 +41,10 @@ The API is meant to enable customers with stronger audit requirements than the a
 (from _metadata_-only level, over _request payloads_ to _request and response payload_ level) of audit
 logs, accepting the increased resource consumption of the API servers.
 
-This API is intentionally **not about filtering events**. Filtering is to be done via an external mechanism (post-filtering).
-It was proven through performance tests (compare alternatives section) that this trade-off is acceptable with small two-digit percent overhead.
+This API is intentionally **very limited in filtering events**. More advanced filtering is to be done via an external mechanism (post-filtering).
```

Comment:
maybe we could explain the post filtering more, like mentioning jq options? Your call

Reply:
added a link to https://docs.openshift.com/container-platform/4.7/security/audit-log-view.html#security-audit-log-basic-filtering_audit-log-view. @bergerhoffer did a great job describing that in our docs.